docs/UserGuide.md

```diff
@@ -45,142 +38,291 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
+>   * Tasks added without specifying a deadline will be displayed under "No Deadline".
+>   * Date formats can be flexible. The application is able to parse commonly-used human-readable date formats.
+>     * e.g. `Monday`, `next wed`, `tomorrow`, `5 days after`, `4 Apr` etc.
+> * Dates can include time as well.
+>   * If only time is specified, it will default to today's date.
+>   * If time is not specified, it will default to the current time of the particular date.
+>   * Time formats are flexible as well. The application supports 24 hour format and AM/PM format.
+>     * e.g. `Monday 3pm`, `today 1930`, `5:30pm`, `10.00 am`
+> * Recurring task will have the same deadline if `daily/weekly/monthly/yearly` is used
+> * Tasks can have any number of label name. (including 0).
+> * Using the `add` command without specifying `task` will interpret the command as `add task`.
 
 Examples:
 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
+ * `add CS2106 Mid terms`
+ * `add CS2103 V0.0 by tmr`
+ * `add task Make baby by next wednesday`
+ * `add task Go to school repeat daily`
+
+### 2.3 Listing all tasks : `list`
+
+Shows a list of all tasks in DoOrDie. Able to filter by type of task (task), or based on status of task such as completed or outstanding.
+
+Format: `list [TYPE]`
+
+> Valid parameters for TYPE:
+> * `tasks` / `task`
+> * `complete` / `completed`
+> * `incomplete` / `outstanding`
+> * `overdue` / `over`
+> * `by DATE`
+> * `from STARTDATE to ENDDATE`
+> * `bookings` / `booking`
+
+Examples:
+
+* `list`<br />
+ Lists all tasks.
+
+* `list overdue tasks`<br />
+ Lists all overdue tasks
+
+* `list outstanding tasks`<br />
+ Lists all outstanding tasks
+
+* `list completed tasks`<br />
+ Lists all completed tasks
+
+* `list by today`<br />
+ Lists all tasks due by today
+
+* `list from monday to friday`<br />
+ Lists all tasks due within Monday-Friday
 
-### 2.3. Listing all persons : `list`
+* `list bookings`<br />
+ Lists all unconfirmed tasks with their respective bookings
 
-Shows a list of all persons in the address book.<br>
-Format: `list`
+### 2.4. Editing a task : `update`
 
-### 2.4. Editing a person : `edit`
+Edits the specified task's details.
 
-Edits an existing person in the address book.<br>
-Format: `edit INDEX [NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Format:`update [TASK_ID] [TASKNAME] [label LABEL] ([(by|on) DATE] | [from START_DATE to END_DATE]) [done]`
 
-> * Edits the person at the specified `INDEX`.
-    The index refers to the index number shown in the last person listing.<br>
+> * Edits the task with the `TASK_ID`
+    The index refers to the id of the task.<br />
     The index **must be a positive integer** 1, 2, 3, ...
-> * At least one of the optional fields must be provided.
+> * At least one of the optional fields excluding `TASK_ID` must be provided.
```

Comment:
Optional, because if the user SELECTS a task, then we will execute on that task. Thus, not requiring an ID